examples/wmt15/config_large.yaml

```diff
@@ -0,0 +1,9 @@
+input_features:
+  - name: en
+    type: text
+    encoder: bert
```

Comment:
what about using GPT2 if it's computationally comparable? Just so that we have a different encoder example

Reply:
Done.